packages/foreman/foreman-release/foreman-release.spec

```diff
@@ -12,11 +12,11 @@
 %define repo_dir %{_sysconfdir}/yum.repos.d
 
 %if 0%{?rhel}
-%define repo_dist el%{rhel}
+%define repo_dist el$releasever_major
```

Comment:
I think we should at least align both packages on `%{dist}`: https://github.com/theforeman/foreman-packaging/pull/10952